etcdmain/config.go

```diff
@@ -41,6 +41,17 @@ const (
 	clusterStateFlagExisting = "existing"
 
 	defaultName = "default"
+
+	// maxElectionMs specifies the maximum value of election timeout.
+	//
+	// A reasonable round-trip time for the continental united states is
```

Comment:
I think we should move this out of our code to our documentation. Our user should see it and understand it.